test/parallel/test-debugger-heap-profiler.js

```diff
@@ -23,11 +24,11 @@ const filename = tmpdir.resolve('node.heapsnapshot');
       await cli.waitForInitialBreak();
       await cli.waitForPrompt();
       await cli.command('takeHeapSnapshot()');
-      JSON.parse(readFileSync(filename, 'utf8'));
+      assert.ok(fs.existsSync(filename));
```

Comment:
```suggestion
      JSON.parse(fs.readFileSync(filename));
```

Reply:
It does not change anything. `toString()` is called on the input.